src/Infrastructure.EntityFramework/Repositories/DatabaseContext.cs

```diff
@@ -108,6 +109,7 @@ protected override void OnModelCreating(ModelBuilder builder)
         var eSsoConfig = builder.Entity<SsoConfig>();
         var eTaxRate = builder.Entity<TaxRate>();
         var eUser = builder.Entity<User>();
+        var eUserSignatureKeyPair = builder.Entity<UserSignatureKeyPair>();
```

Comment:
🎨 We have been trying lately to move these customizations and defaults out to specific entity type config classes.